docs/docs/api/Authentication/login.md

```diff
@@ -11,46 +11,41 @@ color: '#E83E8C',
 
 ```json
 {
-    "Username": string,
-    "Password": string,
+    "email": string,
+    "password": string,
 }
 ```
 
 ### Example Request
+
 #### Body
+
 ```json
 {
-    "Username": "Chris",
-    "Password": "password"
+  "email": "nestor@gmail.com",
+  "Password": "abc123"
 }
 ```
+
 ### Possible Responses
+
 #### Immediate Success
+
 ```json
 {
-	"code": 1,
-	"message": "successfully logged in",
-	"data": {
-	    "token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE1OTU4MjQyNzUsImlhdCI6IjIwMjAtMDctMjdUMDA6MjY6MTUuNzg5NTg0Mi0wNDowMCIsInN1YiI6ImNocmlzIn0.5US2_ITKcfgkpEbfsR-gxXbGPFY6XsgJPcGA5qaBD1M"
-	}
+  "token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE1OTU4MjQyNzUsImlhdCI6IjIwMjAtMDctMjdUMDA6MjY6MTUuNzg5NTg0Mi0wNDowMCIsInN1YiI6ImNocmlzIn0.5US2_ITKcfgkpEbfsR-gxXbGPFY6XsgJPcGA5qaBD1M"
 }
 ```
+
 **Note**: If you log in again (while you're already logged in), any previous **access & refresh tokens** will be revoked. <br></br>
 
 **Note**: The access token will be returned in `data.token`. The refresh token will be returned as an HTTPOnly, Secure, SameSite Cookie.
```

Comment:
Similar to the comment above, this can be removed as well.

Reply:
Fixed: Changes are reflected in recent commits